cartography/intel/aws/s3.py

```diff
@@ -70,16 +70,22 @@ def get_policy(bucket: Dict, client: botocore.client.BaseClient) -> str:
     """
     Gets the S3 bucket policy. Returns policy string or None if no policy
     """
+    policy = ""
```

Comment:
I think this could be problematic because this will eventually get passed to `parse_policy` https://github.com/lyft/cartography/blob/7ed49ea0509b3e41a81ead06b53a2886947f8f8a/cartography/intel/aws/s3.py#L263 and `parse_policy` expects either `None` or a `Dict`.